assets/js/payplug-admin-bancontact.js

```diff
@@ -23,8 +23,6 @@
 		},
 		checkbancontact: (event)=> {
 
-			payplug_admin.disableBancontact();
```

Comment:
is this in the task scope?

Reply:
PM asked for it in order to have all payment methods having the same behaviour